src/treeCss.css

```diff
@@ -26,6 +26,17 @@
   padding-left: 5px;
   padding-right: 5px;
   border-radius: 0px 0px 3px 3px;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

Comment:
So I got annoyed that text was selected when I tried to make selections. Not sure if we should keep this. Again need to ask UX here I guess

Reply:
yea we generally don't allow text selection, so this should be fine